cloud/shared/bin/destroy.py

```diff
@@ -0,0 +1,14 @@
+#! /usr/bin/env python3
```

Comment:
You don't need shebang here given that this file is not main (and won't be run from command line)
https://google.github.io/styleguide/pyguide.html#37-shebang-line